src/modules/75-cf/pages/environments/EnvironmentsPage.tsx

```diff
@@ -52,17 +51,14 @@ export const NameCell = withEnvironment(({ environment }) => {
   const { getString } = useEnvStrings()
   return (
     <Layout.Horizontal
-      className={css.mediumMaxWidth}
       flex={{ distribution: 'space-between', align: 'center-center' }}
       padding={{ left: 'small', right: 'small' }}
     >
-      <Layout.Vertical className={css.calcMaxWidth} spacing="xsmall">
+      <Layout.Vertical spacing="xsmall">
         <Text style={{ color: Color.BLACK, font: FontVariation.BODY2 }}>{environment.name}</Text>
 
         {environment.description && (
-          <Container>
-            <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
-          </Container>
+          <Text style={{ font: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
```

Comment:
Don't know how I didn't spot this before, but for this (and the one above), we need to use the `font` prop, not the `style` prop for variations and colour
```suggestion
          <Text font={{ variation: FontVariation.SMALL, color: Color.GREY_600 }}>{environment.description}</Text>
```

Reply:
Don't think color is a prop of font?

Reply:
```suggestion
          <Text color={Color.GREY_600} font={{ font: FontVariation.SMALL }}>{environment.description}</Text>
```

Reply:
Lovely thank you have changed